pftools/python/setup.py

```diff
@@ -7,7 +7,7 @@
 
 setup(
     name='pftools',
-    version="1.3.6",
+    version="1.3.6.1",
```

Comment:
This is an invalid version number. You need to go to `1.3.7` or even `1.4.0` since it is a new feature rather than a fix.

Reply:
Actually reading the spec, it should not matter.

Reply:
thanks @jourdain fixed